spec/features/access_points/digital_collections_spec.rb

```diff
@@ -17,4 +17,23 @@
       expect(page).to have_css('a', text: 'Submit materials')
     end
   end
+
+  it 'has the facet options for digital collections' do
+    within('.facet-options') do
+      expect(page).to have_text('Show collections only')
+      expect(page).to have_text('Show collections and individual items')
+    end
+  end
+
+  it 'Toggles the radio buttons' do
+    expect(page).to have_css('article', count: 7)
+    expect(page).to have_css('.filter-value', text: 'Digital Collection')
+
+    choose "Show collections and individual items"
+    expect(page).to have_css('.filter-value', text: 'Stanford Digital Repository')
+    expect(page).to have_css('article', count: 1)
```

Comment:
The language of these buttons, 'Show collections only' and 'Show collections and individual items' makes it seem like the latter would be additive. So going from 7 to 1 doesn't really make sense on the surface. I understand it makes sense with the underlying facets, just saying it is a little counterintuitive.

Reply:
I'm hoping it makes more sense with real-world data 🤞